schemas/core/documents/giftCardsCollection/index.ts

```diff
@@ -18,6 +18,11 @@ export default {
       type: "string",
       name: "description",
     },
+    {
+      title: "Font Color",
+      name: "fontColor",
+      type: "color",
```

Comment:
Add a description property with the correct information here.

Reply:
okay